web/l10n/de.json

```diff
@@ -144,6 +144,8 @@
     "commandData": "Daten",
     "commandPhone": "Rufnummer",
     "commandMessage": "Nachricht",
+    "commandTimezoneOffset": "Zeitverschiebung zu UTC",
```

Comment:
The way it works is that you only add/modify English strings, then Transifex automatically picks up the file directly from the repository and people can translate it. Then with every new release I update localization files using special script.

You can just revert it.